src/Command/Debug/NamespacesPrinter.php

```diff
@@ -79,22 +63,45 @@ protected function execute (InputInterface $input, OutputInterface $output)
             $io->note("Warning:\nThere are multiple namespaces pointing to the same directory.\nTry to reuse existing namespaces before creating new ones.");
         }
 
-        return 0;
+        return true;
+    }/**
```

Comment:
CS: The indentation is a bit off. Can you also add a few newlines here?

Reply:
Oh yeah, that is broken.